tutorials/backend/hasura-v3-ts-connector/tutorial-site/content/predicates.md

```diff
@@ -0,0 +1,352 @@
+---
+title: "Predicates"
+metaTitle: 'Predicates | Hasura DDN Data Connector Tutorial'
+metaDescription: 'Learn how to build a data connector for Hasura DDN'
+---
+
+[![Predicates in the connector video](https://img.youtube.com/vi/-2URtvnYi3o/0.jpg)](https://www.youtube.com/watch?v=-2URtvnYi3o)
+
+[_**Click here to watch the video**_](https://www.youtube.com/watch?v=-2URtvnYi3o)
+
+[//]: # (TODO - embed player?)
+
+So now we've set up a basic data connector for a SQLite database running locally. Now we'll start to implement 
+predicates by turning them into where clauses in the generated SQL.
+
+Predicates in GraphQL are expressions which determine the conditions under which data is retrieved or manipulated. 
+For example: a `where` clause.
+
+## Where Clause
+
+Let's pick up from where we left off. We can modify our SQL template in our `fetch_rows` function to include a `WHERE 
+clause`:
+
+```typescript
+const sql = `SELECT ${fields.join(", ")} FROM ${request.collection} ${where_clause} ${limit_clause} ${offset_clause}`;
+```
```

Comment:
As an earlier comment suggested: let's include the entire function and use syntax highlighting to illustrate what the new `sql` should be set equal to.

Reply:
Syntax highlighting not supported in this infra yet